package.json

```diff
@@ -7,21 +7,33 @@
   "homepage": "https://algorithmia.com/",
   "license": "The MIT License (MIT)",
   "scripts": {
-    "start": "NODE_ENV=production node ./server/index.js",
+    "apidocs:build": "cd api-docs && bundle exec rake build",
+    "apidocs:setup": "git submodule init && git submodule update && cd api-docs && rvm install 2.3.0 && gem install bundle && bundle install",
```

Comment:
I got an error on this command when getting set up:
```
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems.rb:241:in `bin_path': can't find gem bundler (>= 0.a) (Gem::GemNotFoundException)
        from /usr/local/bin/bundle:22:in `<main>'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! algorithmia-dev-center@1.0.0 apidocs:setup: `git submodule init && git submodule update && cd api-docs && rvm install 2.3.0 && gem install bundle && bundle install`
```
What's interesting is that running each command individually worked fine.

Reply:
Agh, for some reason I'm not able to repro 😬are you still seeing that when you run the `setup` script?